Core/Core/UIViews/HorizontalMenuViewController.swift

```diff
@@ -18,7 +18,7 @@
 
 import UIKit
 
-open class HorizontalMenuViewController: UIViewController {
+open class HorizontalMenuViewController: ScreenViewLoggerViewController {
```

Comment:
Seems unnecessary.

Reply:
We need this because `SyllabusTabViewController` already inherits from `HorizontalMenuViewController`.

Reply:
Ah ok, my bad.